src/main/java/ua/vedroid/cinema/config/AppConfig.java

```diff
@@ -0,0 +1,50 @@
+package ua.vedroid.cinema.config;
+
+import java.util.Properties;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:db.properties")
+@ComponentScan(value = {
+        "ua.vedroid.cinema.service",
+        "ua.vedroid.cinema.dao"
```

Comment:
`AuthenticationServiceImpl` is work? Because you didn't add the directory `security`  in this place, soo I'm not sure.